src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1423,7 +1423,19 @@ public override void VisitLocalFunction(ILocalFunctionOperation operation)
             LogString(")");
             LogCommonPropertiesAndNewLine(operation);
 
-            Visit(operation.Body);
+            if (operation.Body != null)
+            {
+                if (operation.IgnoredBody != null)
+                {
+                    Visit(operation.Body, "Body");
+                    Visit(operation.IgnoredBody, "IgnoredBody");
+
+                }
+                else
+                {
+                    Visit(operation.Body);
+                }
+            }
```

Comment:
>} [](start = 12, length = 1)

Consider asserting that peration.IgnoredBody is null in the else, to complete validation.